### PR TITLE
feat: add accessories

### PR DESCRIPTION
**Pull request by scriptswhisper**
# Pull Request
Compile all the information written below to send a legitimate pull request to the repository!

### This PR is labeled as:
*Choose only one and remember to select the label even on the right.*

- [x] Feature
- [ ] Bug
- [ ] Other

### This PR has the following linked issues:
- Doesn't close any issue.

## Solution
Adds Steam Deck accessories and one useful website for games compatibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ Choose what you find more suitable for you:
...
+- [NexDock](https://nexdock.com/features/) - Turn Steam Deck into a laptop.
+- [Killswitch](https://dbrand.com/killswitch) - Protective case made by dbrand for Steam Deck.
+- [Carrying Case](https://jsaux.com/products/upgraded-carrying-case-for-steam-deck-bg0106) - Protective case for Steam Deck for accessories, chargers and cables.
+- [Standing Case](https://jsaux.com/products/upgraded-protective-standing-case-for-steam-deck-pc0105) - Protective Standing Case for Steam Deck.
+- [Silicon Case](https://jsaux.com/products/steam-deck-protective-case) - Silicon Protective Case for Steam Deck.
+- [6-in-1 Docking Station](https://jsaux.com/products/upgraded-docking-station-for-steam-deck) - JSAUX 6-in-1 Docking Station for Steam Deck.
+- [Steam Deck Dock](https://store.steampowered.com/steamdeckdock) - Steam Deck Docking Station by Valve.
+- [iVoler Dock](https://www.ivoler.com/it-it/products/docking-station-per-steam-deck-ivoler-6-in-1-alluminio-steam-deck-dock-supporto-hub-usb-c-con-hdmi-2-0-4k-60hz-ethernet-1000m) - iVoler 6in-1 Docking Station for Steam Deck.
```
[the remark this answers is not in the document]
Comment (scriptswhisper) [reply README.md]: Replaced with english version

File: README.md [hunk 2]
```diff
@@ -58,6 +58,20 @@ Choose what you find more suitable for you:
...
+- [Syntech Dock](https://syntechhome.com/products/syntech-docking-station-compatible-with-steam-deck) - Syntech 6-in-1 Docking Station for Steam Deck.
+- [UGREEN Dock](https://eu.ugreen.com/collections/docking-station/products/copy-of-snap-deal-ugreen-docking-station-for-steam-deck) - UGREEN Docking Station for Steam Deck.
+- [Extreme Rate Shells](https://extremerate.com/collections/steam-deck-shells) - Extreme Rate Steam Deck Shells.
+- [GamingCobra Shells](https://www.gamingcobra.com/collections/steam-deck-shells) - Gaming Cobra Steam Deck Shells.
+- [JSAUX Transparent Back Plate](https://jsaux.com/products/transparent-back-plate-for-steam-deck-pc0106) - JSAUX Transparent Back Plate for Steam Deck.
+- [DEVASO Transparent Back Plate](https://www.amazon.com/dp/B0C2T6KGLF) - DEVASO Transparent Back Plate for Steam Deck.
```
[the remark this answers is not in the document]
Comment (scriptswhisper) [reply README.md]: Agree!

File: README.md [hunk 3]
```diff
@@ -58,6 +58,19 @@ Choose what you find more suitable for you:
...
 - [JSAUX Cooling Fans](https://jsaux.com/products/fan-cooler-for-steam-deck-gp0200) - Cooling fans for reducing overall temperatures made by JSAUX.
+- [NexDock](https://nexdock.com/features/) - Turn Steam Deck into a laptop.
+- [Killswitch](https://dbrand.com/killswitch) - Protective case made by dbrand for Steam Deck.
+- [Carrying Case](https://jsaux.com/products/upgraded-carrying-case-for-steam-deck-bg0106) - Protective case for Steam Deck for accessories, chargers and cables.
+- [Standing Case](https://jsaux.com/products/upgraded-protective-standing-case-for-steam-deck-pc0105) - Protective Standing Case for Steam Deck.
+- [Silicon Case](https://jsaux.com/products/steam-deck-protective-case) - Silicon Protective Case for Steam Deck.
+- [6-in-1 Docking Station](https://jsaux.com/products/upgraded-docking-station-for-steam-deck) - JSAUX 6-in-1 Docking Station for Steam Deck.
+- [Steam Deck Dock](https://store.steampowered.com/steamdeckdock) - Steam Deck Docking Station by Valve.
+- [iVoler Dock](https://www.ivoler.com/products/upgraded-docking-station-for-steam-deck) - iVoler 6in-1 Docking Station for Steam Deck.
```
Comment (airscripts) [inline README.md:68]: Unfortunately this link is not working for me...it redirects back to it-it page.  
Is the same for you?

**Review by airscripts — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (scriptswhisper) [reply README.md:68]: I've just tried in incognito and with another browser but points to the right product in En [screen here](https://i.imgur.com/ewxsp7j.jpg). Could you please try it again?

Comment (airscripts) [reply README.md:68]: I've tried with another device and different browsers, same result.
I think it's something related to my ISP, so I'll mark this as legitimate!

Comment (airscripts) [inline README.md:68]: Nicely done!

**Review by airscripts — APPROVED** [review]

**Comment by airscripts** [comment]
Nicely done!

**Pull request merged**